packages/plugin-rename-all-fields/src/types.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Handler to rename collection.
+ * Example: snakeToCamelCase, snakeToPascalCase, ...
+ */
+export type Options = (string) => string;
```

Comment:
can be async ?

Reply:
No, I don't see any use case but it's not costly to do. Do you see one ?

Reply:
🤷‍♂️

Reply:
i'm just thinking about like you say it cost nothing IMO and it cover all use case if there is

Reply:
Done